src/pages/blog/typography.md

```diff
@@ -116,7 +116,7 @@ ___
 
 ### Image
 
-![Minion](http://octodex.github.com/images/minion.png)
+![Minion](https://octodex.github.com/images/minion.png)
```

Comment:
Thanks for this! The Netlify build kept warning me about it but I was too lazy to fix it 😅

Reply:
😆yeap that was the only thing.